src/core/io/src/4C_io_input_spec.hpp

```diff
@@ -94,6 +95,11 @@ namespace Core::IO
      */
     [[nodiscard]] const Internal::InputSpecImpl& impl() const;
 
+    /**
+     * Get the name of this InputSpec. The name is the identifier used in the input file.
+     */
+    [[nodiscard]] std::string name() const;
```

Comment:
```suggestion
    [[nodiscard]] const std::string& name() const;
```

Actually, `std::string_view` would be even better, but 4C is generally quite std::string reliant, so let's stick with this.